cmd/rookd/api.go

```diff
@@ -31,26 +31,30 @@ import (
 )
 
 var apiCmd = &cobra.Command{
-	Use:    "api",
-	Short:  "Runs the Rook REST API service",
-	Hidden: true,
+	Use:   "api",
+	Short: "Runs the Rook API service",
 }
 var apiPort int
+var repoPrefix string
 
 func init() {
-	apiCmd.Flags().IntVar(&apiPort, "api-port", 0, "port on which the api is listening")
+	apiCmd.Flags().IntVar(&apiPort, "port", 0, "port on which the api is listening")
+	apiCmd.Flags().StringVar(&repoPrefix, "repo-prefix", "quay.io/rook", "the repo from which to pull images")
+	apiCmd.Flags().StringVar(&cfg.containerVersion, "container-version", "latest", "version of the rook container to launch")
```

Comment:
Instead of container-version, i would call it `tag` or `version-tag`. It is more correct for container images.